h_transport_materials/materials/lipb.py

```diff
@@ -0,0 +1,245 @@
+    author="schumacher",
+    year=1990,
+    isotope="H",
+    units="m-3 Pa-1/2",
+)
+
+
+reiter_src = "F. Reiter, DOI:10.1016/0920-3796(91)90003-9"
+reiter_diffusivity_h = ArrheniusProperty(
+    pre_exp=4.03e-08 * atom_density_lipb,
+    act_energy=19500 * k_B / Rg,
+    range=(508, 700),
+    source=reiter_src,
+    name="H Reiter (1991)",
+    author="reiter",
+    year=1991,
+    isotope="H",
+)
+reiter_diffusivity_d = ArrheniusProperty(
+    pre_exp=4.03e-08 * atom_density_lipb,
+    act_energy=19500 * k_B / Rg,
+    range=(508, 700),
+    source=reiter_src,
+    name="D Reiter (1991)",
+    author="reiter",
+    year=1991,
+    isotope="D",
+)
+reiter_diffusivity_t = ArrheniusProperty(
+    pre_exp=4.03e-08 * atom_density_lipb,
+    act_energy=19500 * k_B / Rg,
+    range=(508, 700),
+    source=reiter_src,
+    name="T Reiter (1991)",
+    author="reiter",
+    year=1991,
+    isotope="T",
+)
+reiter_solubility_h = Solubility(
+    pre_exp=2.44e-08 * atom_density_lipb,
+    act_energy=1350 * k_B / Rg,
+    range=(508, 700),
+    source=reiter_src,
+    name="H Reiter (1991)",
+    author="reiter",
+    year=1991,
+    isotope="H",
+    units="m-3 Pa-1/2",
+)
+reiter_solubility_d = Solubility(
+    pre_exp=2.36e-08 * atom_density_lipb,
+    act_energy=1350 * k_B / Rg,
+    range=(508, 700),
+    source=reiter_src,
+    name="D Reiter (1991)",
+    author="reiter",
+    year=1991,
+    isotope="D",
+    units="m-3 Pa-1/2",
+)
+reiter_solubility_t = Solubility(
+    pre_exp=2.32e-08 * atom_density_lipb,
+    act_energy=1350 * k_B / Rg,
+    range=(508, 700),
+    source=reiter_src,
+    name="T Reiter (1991)",
+    author="reiter",
+    year=1991,
+    isotope="T",
+    units="m-3 Pa-1/2",
+)
+
+
+aiello_src = (
+    "A. Aiello, A. Ciampichetti, G. Benamati, DOI:10.1016/j.fusengdes.2005.06.364"
+)
+aiello_solubility = Solubility(
+    pre_exp=4.66e-06 * atom_density_lipb,
+    act_energy=13399 * k_B / Rg,
+    range=(600, 900),
+    source=aiello_src,
+    name="H Aiello (2006)",
+    author="aiello",
+    year=2006,
+    isotope="H",
+    units="m-3 Pa-1/2",
+)
+
+
+shibuya_src = (
+    "Y. Shibuya, M. Aida, Y.Fujii, M. Okamoto, DOI:10.1016/0022-3115(87)90006-7"
+)
+shibuya_diffusivity = ArrheniusProperty(
+    pre_exp=2.62e-03 * atom_density_lipb,
+    act_energy=6630 * k_B / Rg,
+    range=(573, 773),
+    source=shibuya_src,
+    name="T Shibuya (1987)",
+    author="shibuya",
+    year=1987,
+    isotope="T",
+)
+
+
+terai_src = (
+    "T. Terai, S. Nagai, T. Yoneoka, Y. Takashi, DOI:10.1016/0022-3115(92)90504-E"
+)
+terai_diffusivity = ArrheniusProperty(
+    pre_exp=2.50e-07 * atom_density_lipb,
+    act_energy=27000 * k_B / Rg,
+    range=(573, 973),
+    source=terai_src,
+    name="T Terai (1987)",
+    author="terai",
+    year=1987,
+    isotope="T",
+)
+
+alberro_src = "G. Alberro, I. Penalva, F. Legarda, G.A. Esteban, DOI:10.1016/j.fusengdes.2015.05.060"
+alberro_solubility = Solubility(
+    pre_exp=5.203e21,
```

Comment:
```suggestion
    pre_exp=8.64e-3*avogadro_nb,
```

I'd like to have the things here as written in the paper for clarity sake